spec/unit/installer/pod_source_installer_spec.rb

```diff
@@ -62,6 +62,30 @@ module Pod
         UI.warnings.should.include 'uses the unencrypted \'git\' protocol'
       end
 
+      it 'does not warn for local repositories with spaces' do
+        @spec.source = { :git => '/Users/kylef/Projects X', :tag => '1.0' }
+        dummy_response = Pod::Downloader::Response.new
+        Downloader.stubs(:download).returns(dummy_response)
+        @installer.install!
+        UI.warnings.length.should.equal(0)
+      end
+
+      it 'does not warn for SSH repositories' do
+        @spec.source = { :git => 'git@bitbucket.org:kylef/test.git', :tag => '1.0' }
+        dummy_response = Pod::Downloader::Response.new
+        Downloader.stubs(:download).returns(dummy_response)
+        @installer.install!
+        UI.warnings.length.should.equal(0)
+      end
+
+      it 'does not warn for SSH repositories on Github' do
+        @spec.source = { :git => 'git@github.com:kylef/test.git', :tag => '1.0' }
+        dummy_response = Pod::Downloader::Response.new
+        Downloader.stubs(:download).returns(dummy_response)
+        @installer.install!
+        UI.warnings.length.should.equal(0)
```

Comment:
This seems identical to the test above - is there expected to be different behavior if the URL host is `github.com`?

Reply:
How would `git@github.com:kylef/test.git` and `git@bitbucket.org:kylef/test.git` differ?

Reply:
Well not so much actually...I used the same tests that were applied here https://github.com/CocoaPods/Core/pull/84/files

Reply:
Huh, that's interesting..

Reply:
@amorde overall harmless I say :) but you were right to comment on it.